app/assets/stylesheets/config/_variables.scss

```diff
@@ -94,4 +94,10 @@
 
   --lh-tight: 1.25; // 20px for 16px font-size
   --lh-base: 1.5; // 24px for 16px font-size
+
+  ////////////////////////////////////////////////////////////////////////////////////
+
+  // Podcasts
+
+  --podcast-spinning-animation: spin 20s linear infinite;
```

Comment:
Can we keep this one outside of `config/_variables.scss`? This file is more for _very global_ variables and this particular variable feels like something very specific...

Reply:
Quick related questions:
- If I was to add a common CSS class that would handle the spin animation between the two places it's used, would the liquid tag of existing articles be updated with the new class?
- How do you manage migration for updates to liquid tags?
- Since the processed HTML of an article is stored in the database, does it ever get processed again?

Thanks!

Reply:
Actually, that was a question for another time. The liquid tag doesn't need an update for this right now.